packages/xo-server/src/decorators.spec.js

```diff
@@ -19,21 +19,24 @@ describe('debounce()', () => {
   })
 
   it('works', done => {
+    const now = Date.now()
+    const mockDate = jest.fn()
+    mockDate.mockReturnValueOnce(now)
+    Date.now = mockDate
```

Comment:
I don't think it's a right approach, I'm even amazed that it works :stuck_out_tongue: 

We should probably use [timer mocks](https://facebook.github.io/jest/docs/en/timer-mocks.html).

Reply:
I tried a bit today, I doesn't appear that timers mocks mess with Date.now(): https://github.com/facebook/jest/blob/master/packages/jest-util/src/fake_timers.js

I see a few references to mocking Date.now in jest's repository itself: https://github.com/facebook/jest/search?utf8=%E2%9C%93&q=Date.now&type=